src/main/java/de/eddyson/tapestry/react/services/nashorn/NashornModuleLoader.java

```diff
@@ -0,0 +1,70 @@
+package de.eddyson.tapestry.react.services.nashorn;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.script.Bindings;
+import javax.script.ScriptEngine;
+import javax.script.ScriptException;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.tapestry5.internal.services.assets.ResourceChangeTracker;
+import org.apache.tapestry5.ioc.Resource;
+import org.apache.tapestry5.services.AssetSource;
+import org.apache.tapestry5.services.assets.StreamableResource;
+import org.apache.tapestry5.services.assets.StreamableResourceProcessing;
+import org.apache.tapestry5.services.assets.StreamableResourceSource;
+import org.apache.tapestry5.services.javascript.ModuleManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Nashorn module loader get's inserted into JavaScript executed by Nashorn and
+ * loads AMD modules from classpath
+ *
+ * @author Felix Gonschorek
```

Comment:
If you don't insist, I'd like you to remove the `@author` JavaDoc. I find git history to be much more meaningful.